pom.xml

```diff
@@ -52,14 +53,57 @@
 		<url>http://github.com/jenkinsci/matlab-plugin</url>
 	  <tag>HEAD</tag>
   </scm>
-
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.jenkins.tools.bom</groupId>
+            <artifactId>bom-2.164.x</artifactId>
+            <version>4</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+</dependencyManagement>

```

Comment:
The indentation on this file is off. In this section and below.

Reply:
Done.